### PR TITLE
Library

### PR DESCRIPTION
**Pull request by sumon0002001**
-Library in JavaScript is a practice project, which includes the user input functionality and managing the dynamically DOM manipulation.
- I used Javascript, HTML5, and CSS3, to implement it.
- **Built With**
- HTML5
- CSS3
- JavaScript

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @sumon0002001, A big heartfelt congratulations for reaching the JS module. 🍾 🍾 

**Your project is looking great, a lot of effort must need for that.**

## The highlights: 🌟 🌟 

* Good PR descriptions and title. 💯 
* The checking for if the book is already the library array is nice. 👍 
*  Using CSS custom properties for color is very nice.👍 

## What to do next: 

**Please make sure all the required point added to get approval**

**[REQUIRED]** 👇 

- [x] In your `README`, in the `Setup` section add instructions for how to run your code locally.
- [x]  Make your commit messages more descriptive, try to explain the work done in that commit shortly.
- [x] Please don't make any changes in the `linters` files, If you have any errors, try to solve them.
- [x] Don't use `Window.alert()` or `alert()`.
- [x] Add `\node_module` in the `.gitignore` file.
- [x]  As you are not using `webpack` so you don't have the package.json file, so please remove the package-lock.json file.
- [x] In the stylesheet, you are reusing the same codes, it is violating the DRY rule. Please check my comments below.
- Also you can check this [link](http://www.websiteoptimization.com/speed/tweak/groupsel/), to group selectors with the same style definition.
##

**[OPTIONAL]**

- [ ] Consider adding a screen-shot of your project in the README.md file.
##

**Please make the changes and ask for a re-review.**
Feel free to reach me on slack @lamiasristy 🙋
Happy Coding 💻

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  font-family: inherit;
+  font-size: inherit;
+  cursor: pointer;
+}
+
+input {
+  color: inherit;
+  font-family: inherit;
+  font-size: inherit;
+}
```
Comment (LamiaSristy) [inline styles/style.css]: Do not write the same codes, you can make a class for the same properties:
```
 .inherit {
         color: inherit;
         font-family: inherit;
         font-size: inherit;
   }

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+.popup_input:focus {
+  outline: blueviolet;
+}
+
+.popup__is-read {
+  display: flex;
+  justify-content: space-between;
+  width: 47%;
+  padding: 1rem;
```
Comment (LamiaSristy) [inline styles/style.css:207]: Make a class 
``` 
.flex { display: flex}

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+  height: 2rem;
+  margin-top: 1.6rem;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100%;
```
Comment (LamiaSristy) [inline styles/style.css]: Add class
```
 .fixed {position: fixed;}

File: index.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+function addBook(e) {
+  e.preventDefault();
+  if (addToLibrary(getBookFromInput())) {
+    updateBooksGrid();
+    closePopup();
+  } else {
+    alert('This book already exists in your library');
+  }
```
Comment (trekab) [inline index.js]: Avoid using Window.alert(). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/))

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :tada: 
Hi @sumon0002001! Nice job working on the suggested changes from the last review. There are just a few changes for you to implement to complete this project. These include:
- [ ] Please delete the exiting `node_modules` from your repository and add the `.DS_Store` to your `.gitignore` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] Remove the alert() highlighted in the code comment.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by Forison — APPROVED** [review]
# Status::Approved✔️
### Good job on addressing the issues raised in the previous review, kindly go ahead and merge the branch and proceed to the next task😇.
#### Happy coding😎
Addo Forison

**Pull request merged**